front/components/DataSourceViewResourceSelectorTree.tsx

```diff
@@ -120,6 +120,8 @@ function DataSourceViewResourceSelectorChildren({
   }
 
   const isTablesView = viewType === "tables";
+  const isEmptyManagedDataSource =
+    dataSourceView.category === "managed" && nodes.length === 0 && !parentId;
```

Comment:
the `!parentId` is to make sure we are at the root of the datasource ?

Reply:
Yessir !